apps/web/app/[lang]/sites/components/browser/SitesBrowser.tsx

```diff
@@ -71,33 +97,60 @@ export const SitesBrowser = ({
                     }}
                   />
                 </AlgoliaFacetDropdown>
-
                 <AlgoliaSearchStats className="hidden sm:block text-dark text-base text-end font-normal font-['DMSans'] leading-normal me-5 mb-2" />
               </div>
             </AlgoliaSearch>
           </div>
           <div className="block sm:hidden col-span-1 lg:col-span-2">
             <AlgoliaMap
               className="h-[400px] sm:h-full w-full relative"
-              center={[-34.61, -58.37]}
+              center={
+                selectedSite
+                  ? [selectedSite._geoloc.lat, selectedSite._geoloc.lng]
+                  : undefined
+              }
               zoom={12}
               minZoom={4}
               scrollWheelZoom={true}
             />
           </div>
 
-          <div className="w-full sm:h-5/6 relative col-span-1 lg:col-span-1 xl:col-span-1 sm:overflow-y-scroll sm:overflow-auto">
+          <div className="block sm:hidden w-full sm:h-5/6 relative col-span-1 sm:overflow-y-scroll sm:overflow-auto">
             <AlgoliaInfiniteHits
               className="relative"
               hit={SiteItem}
+              browserDispatchAction={
+                sitesBrowserActions.SET_SELECTED_SITE_MODAL
+              }
+              onChange={handleHits}
+            />
+          </div>
+
+          <div className="hidden sm:block w-full sm:h-5/6 relative col-span-1 sm:overflow-y-scroll sm:overflow-auto">
+            <AlgoliaInfiniteHits
+              className="relative"
+              hit={SiteItem}
+              browserDispatchAction={sitesBrowserActions.SET_SELECTED_SITE}
               onChange={handleHits}
             />
           </div>
         </div>
-        <div className="hidden sm:h-[80vh] sm:block col-span-1 lg:col-span-2">
+        {selectedSite && showSitePreview && (
+          <div className="hidden sm:h-[80vh] sm:block m-4 sm:row-start-1 sm:col-start-2 sm:col-span-1 z-20 overflow-y-auto">
+            <SitePreviewCard
+              site={selectedSite}
+              onClose={handleOnClosePreview}
+            ></SitePreviewCard>
```

Comment:
```suggestion
            />
```
Use self closing tags when there is no children